scripts/propose.py

```diff
@@ -0,0 +1,17 @@
+import grpc
+import logging
+from rchain.client import RClient
+from rchain.client import RClientException
+
+root = logging.getLogger()
+root.setLevel(logging.DEBUG)
+error=0
+with grpc.insecure_channel('localhost:40401') as channel:
+	client = RClient(channel)
+	try:
+		client.propose()
+	except RClientException as e:
+		# NoNewDeploys is a valid scenario
+		if "NoNewDeploys" not in str(e):
+			error=1
+print(error)
```

Comment:
Use the logger instead of print.

Here should be `sys.exit(error)`

print a number is not really fancy, logging more info helps debug

Reply:
https://github.com/rchain/rchain-testnet-node/pull/17/commits/70ed739bac26bcb6ff383de9c062cb712b950ef8